src/FixedDataTableContainer.js

```diff
@@ -30,6 +29,7 @@ class FixedDataTableContainer extends React.Component {
     defaultScrollbars: true,
     scrollbarXHeight: Scrollbar.SIZE,
     scrollbarYWidth: Scrollbar.SIZE,
+    shouldUseLegacyComponents: false,
```

Comment:
@pradeepnschrodinger  should it be true or false by default ??

Reply:
This should be false by default to encourage users to NOT rely on legacy components.